confmap/resolver.go

```diff
@@ -10,10 +10,19 @@ import (
 	"regexp"
 	"strings"
 
+	"go.opentelemetry.io/collector/featuregate"
+
 	"go.uber.org/multierr"
 	"go.uber.org/zap"
 )
 
+var EnableMergeAppendOption = featuregate.GlobalRegistry().MustRegister(
+	"exporter.enableMergeAppendOption",
```

Comment:
I would namespace it under confmap
```suggestion
	"confmap.enableMergeAppendOption",
```